x/emissions/proto/emissions/v7/query.proto

```diff
@@ -1592,3 +1611,36 @@ message GetActiveInferersForTopicRequest {
 message GetActiveInferersForTopicResponse {
   repeated string inferers = 1;
 }
+
```

Comment:
consider adding (amino.dont_omitempty) = true otherwise it will remove the parameter if the value is zero (or zero value).